src/Microsoft.Dnx.Runtime/ProjectReader.cs

```diff
@@ -459,6 +459,13 @@ private static CompilerOptions GetCompilationOptions(JsonObject rawObject)
                 return null;
             }
 
+            string keyFile = rawOptions.ValueAsString("keyFile");
+
+            if (keyFile != null)
+            {
+                keyFile = Path.GetFullPath(Path.Combine(projectPath, keyFile));
```

Comment:
I don't think this should happen in the project reader. The reader should contain the raw value